chore: update babel-preset-stage (0,1,2,3 readme files) (#10111)
babel/babel: packages/babel-preset-stage-0/README.md

    plugins: [
      require("@babel/plugin-syntax-dynamic-import"),
      [require("@babel/plugin-proposal-decorators"), { "legacy": true }],
      [require("@babel/plugin-proposal-class-properties"), { "loose": true }],
    ],
    presets: [
      // ...
    ],
  };
};
```

> **NOTE: Compatibility with @babel/plugin-proposal-class-properties**

Reviewer: ```suggestion
**NOTE: Compatibility between `@babel/plugin-proposal-class-properties` and `@babel/plugin-proposal-decorators`**
```

Author: I will apply these changes for all, right?

If you are including your plugins manually and using `@babel/plugin-proposal-class-properties`, make sure that `@babel/plugin-proposal-decorators` comes before `@babel/plugin-proposal-class-properties`.

> When using the `legacy: true` mode, `@babel/plugin-proposal-class-properties` must be used in `loose: true` mode to support the @babel/plugin-proposal-decorators.

Reviewer: ```suggestion
When using the `legacy: true` option of `@babel/plugin-proposal-decorators`, `@babel/plugin-proposal-class-properties` must be used in `loose: true` mode.
```

> If are not using `@babel/plugin-proposal-decorators`, `loose` mode is not needed at all.

Reviewer: I'd remove "at all", since there are many other valid reasons.
